images/namesrv/alpine/Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN set -eux; \
     apk add --virtual .build-deps curl gnupg unzip; \
     curl https://archive.apache.org/dist/rocketmq/${ROCKETMQ_VERSION}/rocketmq-all-${ROCKETMQ_VERSION}-bin-release.zip -o rocketmq.zip; \
     curl https://archive.apache.org/dist/rocketmq/${ROCKETMQ_VERSION}/rocketmq-all-${ROCKETMQ_VERSION}-bin-release.zip.asc -o rocketmq.zip.asc; \
-	curl https://www.apache.org/dist/rocketmq/KEYS -o KEYS; \
+	curl https://downloads.apache.org/rocketmq/KEYS -o KEYS; \
```

Comment:
Could you provide the reason to change it? It seems that this modification is not related to the title

Reply:
this is the result of the old url:
![image](https://user-images.githubusercontent.com/41141842/131794625-95777928-186e-4467-af9e-4c94ef94314a.png)

Reply:
Ok, this is indeed a problem. I think you can use `curl -L` to solve the redirect problem, which is the same as that in rocketmq-docker repo.